src/BlockchainWriter/Controller.ts

```diff
@@ -103,11 +103,13 @@ export class Controller {
 
     const data = poetAnchorToData(poetAnchor)
     const txId = await anchorData(data)
+    const blockInfo: any = await this.bitcoinCore.getBlockchainInfo()
```

Comment:
Can you add an interface to https://github.com/poetapp/node/blob/master/src/Helpers/Bitcoin.ts instead of using `any`?

Reply:
Adding it to bitcoincore.d.ts actually since it defines the return as any.